lib/generators/ops_care/templates/okcomputer.rb

```diff
@@ -5,15 +5,16 @@
 ## Don't edit this file unless you know what you are doing,
 ## it could result in an undeployable app.
 ##
-require 'securerandom'
+require "securerandom"
 
 # If the envvar is not set, generate a random string.
 # It will be inaccessible, but at least secured.
-if ENV['HEALTHCHECK_TOKEN'].blank?
+if ENV["HEALTHCHECK_TOKEN"].blank?
   healthcheck_token = SecureRandom.hex(32)
-  Rails.logger.info "No HEALTHCHECK_TOKEN envvar found. Staring OkComuter with a generated one: #{healthcheck_token}"
+  Rails.logger.info "No HEALTHCHECK_TOKEN envvar found. Staring OkComuter " \
+                    "with a generated one: #{healthcheck_token}"
```

Comment:
I'm not fond of splitting lines because a string is too long.
The point of the 80chars limit is to make sure code is in small, readable chunks.
I thought we said strings were not supposed to be constrained by the 80chars limit  ?

Reply:
Will look at the string length rule in Rubocop and see what we can do there. 

Is there a reason this string isn't added into the `en.yml` file which would be my preference.

Reply:
Looks like there is a typo that needs to be fixed too. 'Starting' instead of 'Staring'.